File: python_solutions/python_solutions/kyu_8/basic_mathematical_operations.py
def basic_op(operator, value1, value2):
    operator_to_function = {
        "*": lambda a,b: a*b,
        "/": lambda a,b: a/b,
        "+": lambda a,b: a+b,
        "-": lambda a,b: a-b,
    }
    return operator_to_function[operator](value1,value2)

File: js/8-kyu/basic-mathematical-operations.js
function basicOp(operation, value1, value2) {
    const operation_to_function = {
        "*": (a, b) => a * b,
        "/": (a, b) => a / b,
        "+": (a, b) => a + b,
        "-": (a, b) => a - b,
    }
    return operation_to_function[operation](value1, value2)
}

const Test = require('@codewars/test-compat');

describe("Tests", () => {
    it("test", () => {
        console.log("Basic tests\n");
        Test.assertSimilar(basicOp('+', 4, 7), 11);
        Test.assertSimilar(basicOp('-', 15, 18), -3);
        Test.assertSimilar(basicOp('*', 5, 5), 25);
        Test.assertSimilar(basicOp('/', 49, 7), 7);
    });
});